modules/indexing/src/test/java/org/apache/ignite/internal/processors/query/oom/DiskSpillingAbstractTest.java

```diff
@@ -488,26 +490,32 @@ protected void checkQuery(Result res, String sql, int threadNum, int iterations)
             watchKey = workDir.register(watchSvc, ENTRY_CREATE, ENTRY_DELETE);
 
             multithreaded(() -> {
-                for (int i = 0; i < iterations; i++) {
-                    IgniteEx grid = fromClient() ? grid("client") : grid(0);
-                    grid.cache(DEFAULT_CACHE_NAME)
-                        .query(new SqlFieldsQuery(sql)
-                            .setLazy(false))
-                        .getAll();
+                try {
+                    for (int i = 0; i < iterations; i++) {
+                        IgniteEx grid = fromClient() ? grid("client") : grid(0);
+                        grid.cache(DEFAULT_CACHE_NAME)
+                            .query(new SqlFieldsQuery(sql)
+                                .setLazy(true))
```

Comment:
maybe we should make laziness as param and run tests in two different configuration, WDYT?

Reply:
It makes sense. We have a nightly suite `IgniteBinaryCacheQueryLazyTestSuite2` where all queries tests are checked with lazy=true settings. For now, I just removed this setter.